decidim-core/spec/controllers/pages_controller_spec.rb

```diff
@@ -36,6 +36,15 @@ module Devise
           expect(response.body).to include(page.content[I18n.locale.to_s])
         end
       end
+
+      context "when a page doesn't exist" do
+        it "redirects to the 404" do
+          get :show, params: { id: "some-page" }
+
+          expect(response.status).to eq(302)
+          expect(response).to redirect_to("/404")
```

Comment:
Now the spec is failing. There's this shared spec to check these situations: https://github.com/decidim/decidim/blob/master/decidim-core/lib/decidim/core/test/shared_examples/errors.rb, but it's for system specs. So either move this to a system spec (probably better), or just assert that the exception is raised (probably easier).

Reply:
I think I'll go with changing the assertion, since in the spec we're changing the Rails config and doing this for a system spec might be quite hard I guess.

Reply:
You mean in the shared spec? Yeah, that config tweak only makes sense for system specs. That's why I suggested either migrating the specs here to system specs, or just changing the assertion, which is definitely easier :+1:.

Reply:
Sorry @deivid-rodriguez, I deleted my messages because I got confused and I didn't realise that the shared spec was for system specs 🤦‍♂️